src/main/java/wooteco/subway/Age.java

```diff
@@ -0,0 +1,34 @@
+package wooteco.subway;
+
+import java.util.Arrays;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public enum Age {
+
+    INFANTS(age -> 1 <= age && age < 6, fare -> 0),
+    CHILDREN(age -> 6 <= age && age < 13, fare -> (int) ((fare - 350) * 0.5)),
+    TEENAGERS(age -> 13 <= age && age <19, fare -> (int) ((fare - 350) * 0.8)),
+    ADULTS(age -> 19 <= age && age < 65, fare -> fare),
+    SENIORS(age -> 65 <= age, fare -> 0),
+    ;
+
+    private final Predicate<Integer> findAgeGroup;
+    private final Function<Integer, Integer> fareCalculator;
```

Comment:
```suggestion
    private final UnaryOperator<Integer> fareCalculator;
```

이런식으로도 작성할 수 있습니다 :)